lib/combo/settings.ts

```diff
@@ -254,6 +254,12 @@ export const SETTINGS = [{
   category: 'entrances',
   type: 'boolean',
   default: false
+}, {
+  key: 'erMmMinorDungeons',
```

Comment:
Should probably be 3 settings, as what's a "Minor Dungeon" in MM is more controversial than in OoT.

We probably want:

 * Shuffle secret shrine
 * Shuffle Ikana Castle
 * Shuffle Beneath the Well